core/unit_test/neighbor_unit_test.hpp

```diff
@@ -316,6 +316,10 @@ void checkFirstNeighborParallelFor( const ListType& nlist,
                                                     num_particle );
     Kokkos::View<int*, memory_space> team_result( "team_result", num_particle );
 
+    Kokkos::deep_copy( N2_result, 0.0 );
+    Kokkos::deep_copy( serial_result, 0.0 );
+    Kokkos::deep_copy( team_result, 0.0 );
```

Comment:
This initialization to 0 is redundant.  These views are initialized upon construction

Reply:
Yeah that's what I figured.